crates/flux-infer/src/fixpoint_encoding.rs

```diff
@@ -1119,6 +1119,19 @@ impl<'genv, 'tcx> ExprEncodingCtxt<'genv, 'tcx> {
             | rty::ExprKind::ForAll(_) => {
                 span_bug!(self.def_span, "unexpected expr: `{expr:?}`")
             }
+            rty::ExprKind::BoundedQuant(kind, lo, hi, body) => {
+                let exprs = (*lo..*hi)
+                    .map(|i| {
+                        let arg = rty::Expr::constant(rty::Constant::from(i));
+                        body.replace_bound_reft(&arg)
+                    })
+                    .collect_vec();
```

Comment:
You don't need to collect into a vector

```suggestion
                    });
```